editor/backend/src/EditCommand.ts

```diff
@@ -23,7 +25,11 @@ export class EditCommand {
   public static commandId = "";
   public static version = "1.0.0";
 
-  public constructor(_arg?: any) { }
+  public iModel: IModelDb;

```

Comment:
Does it make sense to be able to change the iModel after construction, or should this be `readonly`?

Reply:
yes, it should

Reply:
done